include/xaptum-tpm/nvram.h

```diff
@@ -34,6 +35,7 @@ extern "C" {
 #define XTPM_SERVER_ID_HANDLE       0x1410008
 #define XTPM_ROOT_XTTCERT_HANDLE    0x1410009
 
+TPMI_RH_NV_INDEX xtpm_ecdaa_key_handle();
```

Comment:
Should this be `TPM2_HANDLE`, since the ECDAA key isn't an NV value?  (I realize it doesn't matter functionally, since they're all just typedefs of `TPM2_HANDLE`).

Reply:
Yes! Good catch! That was a bad copy-paste

I just pushed this fix